src/utils.ts

```diff
@@ -149,6 +154,9 @@ function getCheckSummary(trivyStatus: number, dockleStatus: number): string {
     summary = `${summary}\n\n${dockleSummary}`;
   }
 
+  let appHyperlink = `<a href=${APP_LINK}>${APP_NAME}</a>`;
+  summary = `${summary}\n\nFor a better experience of handling the detected vulnerabilities and best practice violations, install ${appHyperlink}.`
```

Comment:
Just a thought:
"For a better experience in managing allowedlist, install scanitizer app"